{{ cookiecutter.repo_name }}/README.md

```diff
@@ -1,157 +1,55 @@
 
 {{ cookiecutter.overview }}
 
-## Requirements
+> ℹ️ Where this documentation refers to the **root folder** we mean where this README.md is located.
 
-To run the code in this GitHub repository, please make sure your system meets the following requirements:
+- [Getting started](#getting-started)
+  - [Requirements](#requirements)
+{% if cookiecutter.create_secrets_file == "Yes" %}- [Required secrets and credentials](#required-secrets-and-credentials){% endif -%}
+- [Licence](#licence)
+- [Contributing](#contributing)
+- [Acknowledgements](#acknowledgements)
 
-- Unix-like operating system (macOS, Linux, …);
-- [`direnv`](https://direnv.net/) installed, including shell hooks;
-- [`.envrc`](.envrc) allowed/trusted by `direnv` to use the environment variables - see
-[below](#allowingtrusting-envrc);
-{%- if cookiecutter.create_secrets_file == "Yes" %}
-- If missing, [create a `.secrets` file](#creating-a-secrets-file) to store untracked secrets;{% endif %}
-- Python 3.5 or above; and
-- Python packages installed from the `requirements.txt` file.
+## Getting started
 
-Note there may be some Python IDE-specific requirements around loading environment variables, which are not considered
-here.
+To be added.
 
-### Allowing/trusting `.envrc`
+### Requirements
 
-To allow/trust the [`.envrc`](.envrc) run the `allow` command using `direnv` at the top level of this repository.
+{% if cookiecutter.create_secrets_file == "Yes" %}- A `.secrets` file with the [required secrets and credentials](#required-secrets-and-credentials){% endif -%}
+- [Load environment variables][docs-loading-environment-variables] from `.envrc`
 
-```shell script
-direnv allow
-```
+To be added.
 {% if cookiecutter.create_secrets_file == "Yes" %}
-### Creating a `.secrets` file
-
-Secrets used by this repository can be stored in a `.secrets` file. **This is not tracked by Git**, and so secrets will
-not be committed onto your remote.
-
-In your shell terminal, at the top level of the repository, create a `.secrets` file.
-
-```shell script
-touch .secrets
-```
-
-Open this new `.secrets` file using a text editor, and add any secrets as environmental variables. For example, to add
-a JSON credentials file for Google BigQuery, add the following code to `.secrets`.
-
-```shell script
-export GOOGLE_APPLICATION_CREDENTIALS="path/to/credentials.json"
-```
-{% endif %}
-### Installing Python packages
-
-To install required Python packages via `pip`, first [set up a Python virtual
-environment](#creating-a-python-virtual-environment); this ensures you do not install the packages globally.
+## Required secrets and credentials
 
-Then run the following `make` command at the top level of this repository:
+To run this project, first make sure [you have a `.secrets` file][docs-loading-environment-variables-secrets]. Then,
+add the following lines to it to load secrets and credentials as environment variables:
 
-```shell script
-make requirements
+```shell
+export SOME_VARIABLE=SOME_VALUE
```

Comment:
Is the shell command here to check it has been loaded as an environment variable? Should we make this more explicit then?

Reply:
This is actually an example secret/credential environment variable to add to `.secrets` (once you've created it). I've made this clearer (hopefully) in commit [`e469722`](https://github.com/ukgovdatascience/govcookiecutter/pull/10/commits/e469722f3ccce6def380f081afb41e01598e80b7).